oauth2/oauth2-jose/src/main/java/org/springframework/security/oauth2/jwt/JwtIssuerValidator.java

```diff
@@ -32,20 +30,30 @@ public final class JwtIssuerValidator implements OAuth2TokenValidator<Jwt> {
 
 	private final JwtClaimValidator<Object> validator;
 
+	private boolean allowEmpty;
+
 	/**
 	 * Constructs a {@link JwtIssuerValidator} using the provided parameters
 	 * @param issuer - The issuer that each {@link Jwt} should have.
 	 */
 	public JwtIssuerValidator(String issuer) {
 		Assert.notNull(issuer, "issuer cannot be null");
+		this.allowEmpty = true;
+		this.validator = new JwtClaimValidator<>(JwtClaimNames.ISS,
+				(claimValue) -> (claimValue != null) ? issuer.equals(claimValue.toString()) : allowEmpty);
+	}
 
-		Predicate<Object> testClaimValue = (claimValue) -> (claimValue != null) && issuer.equals(claimValue.toString());
-		this.validator = new JwtClaimValidator<>(JwtClaimNames.ISS, testClaimValue);
+	/**
+	 * Whether to allow the {@code iss} claim to be empty. The default value is
+	 * {@code false}
+	 */
+	public void setAllowEmpty(boolean allowEmpty) {
+		this.allowEmpty = allowEmpty;
 	}
 
 	@Override
 	public OAuth2TokenValidatorResult validate(Jwt token) {
-		Assert.notNull(token, "token cannot be null");
```

Comment:
I think the intent is to say the parameter name, not the datatype. Will you please leave this as-is?